CRM/Admin/Form/MessageTemplates.php

```diff
@@ -182,6 +178,10 @@ public function buildQuickForm() {
       );
     }
 
+    $this->add('textarea', 'msg_text', ts('Text Message'),
+      "cols=50 rows=6"
+    );
+
```

Comment:
It's probably not harmful, but I don't think this change actually does anything to affect the display.